src/Form/index.tsx

```diff
@@ -196,7 +197,9 @@ function Form({
   >({});
   const [, setRepeatChanged] = useState(false);
 
-  const [integrations, setIntegrations] = useState<Record<string, any>>({});
+  const [integrations, setIntegrations] = useState<null | Record<string, any>>(
+    null
```

Comment:
I had to change the initial value to null so we can distinguish between integrations not being set yet vs the form not having integrations